components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -364,6 +364,19 @@ func Run(options ...RunOption) {
 	wg.Wait()
 }
 
+func installDockerConfig() {
+	const dockerConfigPath = "/home/gitpod/.docker/config.json"
```

Comment:
Can we use the HOME variable instead of hardcoding the path here?

Reply:
I am not sure if its set at that stage and have no way to test it.
All other install functions in supervisor.go also hardcode the path.

Reply:
Hmm, np. Other than that, please sign the CLA fast so that we can get this merged! 🚀 😄

Reply:
I signed the CLA on March, 31st.

Reply:
@axonasif what can I help to bring this MR forward?
the helper works. i assume the supervisor works too (but can't test locally)

Reply:
It also looks like we can't reopen the PR anymore. Can you please create a new one?
<img width="481" alt="Screenshot 2022-05-11 at 8 07 05 AM" src="https://user-images.githubusercontent.com/39482679/167754484-d5acd8fd-9997-446e-8d74-0c37bd2cb39c.png">